java/fury-core/src/main/java/org/apache/fury/meta/MetaStringEncoder.java

```diff
@@ -70,6 +70,9 @@ public MetaString encode(String input, Encoding[] encodings) {
   public MetaString encode(String input, Encoding encoding) {
     Preconditions.checkArgument(
         input.length() < Short.MAX_VALUE, "Long meta string than 32767 is not allowed");
+    if (encoding != Encoding.UTF_8 && !isASCII(input)) {
```

Comment:
Just found that we can use `org.apache.fury.serializer.StringSerializer#isLatin` in java instead. `isLatin` is OK too here

Reply:
Got it, PR has been updated. Thanks for the review!